loomi/regulariza-diagonal-client
# Feature/report modal

## Feature
 - Report Modal
---

**What I did:**

- Create Report modal
- Create pickList component
- Create Calendar component

**How to test:**

1. In your browser go to [stage](http://project.netlify.app/) or [localhost](http://localhost:3000/)
2. Check if the component/view looks just like the [mockup](https://xd.adobe.com/) in different screen sizes (width between 1024px and 1920px)
3. Test the things that you think are worth testing, even the ones that are not in this description

![image](https://user-images.githubusercontent.com/32523401/132562080-72cb1a0e-0e74-4458-99d5-dc52d671a2be.png)
